Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1766,8 +1770,6 @@ namespace PlayerCommands
 		{
 			int item = ToInt(GetParam(args, ' ', 2));
 			int money = ToInt(GetParam(args, ' ', 3));
-			int min_stock = ToInt(GetParam(args, ' ', 4));
-			int max_stock = ToInt(GetParam(args, ' ', 5));
 
 			if (money < 1 || money > 1000000000)

```

Comment:
Instead of 1000000000 write 1'000'000'000

Reply:
We're not in Switzerland, Italy or Liechtenstein. If you're going to use thousands separators, at least use a comma or a full stop.

Reply:
Okay, But then it wont compile. The ' is a special character in C++ to allow you split up numbers without blocking other syntax.

Reply:
https://en.cppreference.com/w/cpp/language/integer_literal

Reply:
Ah I see - I assumed your comment was about some user-display code, not about the actual code itself. Makes sense then.